src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: Tutorial 1 Hello World
+---
+
+# Tutorial 1: Hello World
+
+This process recreates the same Hello World sample application available [here](https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld).
+
+## Overview of the parts
+
+This application consists of a solution that contains four projects: SiloHost, OrleansClient, HelloWorld.Interfaces, and HelloWorld.Grains.
+
+```csharp
+[...]
+        private static async Task<ISiloHost> StartSilo()
+        { 
+            // define the cluster configuration 
+            var builder = new SiloHostBuilder()
```

Comment:
I apologize, when I commented about the silo config part, I meant this line, and the lines attached to this line after it... 

I realize it might be confusing....

Same with client config. Did you deleted client config ?